content/en/smart-contracts/developing-contracts/best-practices/index.md

```diff
@@ -65,3 +65,13 @@ Any contracts sending funds to actors that are not native accounts (`f1` or `f3`
 ### Contracts interacting with built-in actors
 
 All contracts interacting with built-in actors must upgrade to the [latest version of Filecoin Solidity library, currently `v0.8`](https://github.com/Zondax/filecoin-solidity/tree/master/contracts/v0.8). The IPLD codec used in the `handle_filecoin_method` solidity entrypoint and the `call_actor` should now be `CBOR (0x51)`, not `DAG_CBOR (0x71)`, as previously used. The underlying encoding (i.e. the payload bytes) are the same, but the codec numbers are now different. `DAG_CBOR` support will be re-enabled in the future but the usage of the codec implies additional runtime guarantees that have not yet been implemented.
+
+## Contract Verification
+
+When deploying contracts to main net, it is important to verify your contracts to improve transparency, security and trustlessness of the network. Verifying your contract involves recompiling your contract's source code to ensure that the produce bytecode, matches your contract's bytecode that is already on the network since your deployment. It is highly recommended for all FVM smart contracts to complete the verification process, soon after deployment. 
```

Comment:
```suggestion
When deploying contracts to mainnet, it is important to verify your contracts to improve transparency, security and trustlessness of the network. The process of verifying your contract involves recompiling your contract's source code to ensure that the produced bytecode matches the bytecode that is already live on the network since it was deployed.

It is highly recommended for all FVM smart contracts to complete the verification process, soon after deployment. 
```